formats/json-tests/commonTest/src/kotlinx/serialization/json/serializers/JsonUnquotedLiteralTest.kt

```diff
@@ -0,0 +1,137 @@
+package kotlinx.serialization.json.serializers
+
+import kotlinx.serialization.builtins.MapSerializer
+import kotlinx.serialization.json.*
+import kotlinx.serialization.test.assertFailsWithSerialMessage
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class JsonUnquotedLiteralTest : JsonTestBase() {
+
+    private fun assertUnquotedLiteralEncoded(inputValue: String) = parametrizedTest { mode ->
+        val unquotedElement = JsonUnquotedLiteral(inputValue)
+
+        assertEquals(
```

Comment:
Given that this assert is not dependent on the mode, it will be executed multiple times without any changes (currently there are 4 modes, IIRC). I suggest just moving it outside of `parametrizedTest` lambda